actors/datacap/tests/datacap_actor_test.rs

```diff
@@ -78,6 +78,8 @@ mod mint {
     #[test]
     fn requires_whole_tokens() {
         let (mut rt, h) = make_harness();
+        rt.skip_verification_expectations_on_drop();
```

Comment:
@anorth The way (I think) reset works is that it changes the reference [here](https://github.com/filecoin-project/builtin-actors/blob/master/runtime/src/test_utils.rs#L162). This leads to the previous object going out of scope and it's `drop` being called and failing. That's why this `skip...` is needed when `reset` is used.
Anyway, I guess I approached it incorrectly i.e I probably should have rewritten the implementation of `reset()` so it works better. Not sure though how to implement it other than changing all fields in `Expectations` to `mut` and resetting them 1 by 1. Wdyt?

Reply:
Can't you just set `skip_verification_on_drop` true inside the `reset()` method, immediately before that line you linked? Or maybe from the Mock.reset()? Then it can go out of scope harmlessly.